lmcache/storage_backend/serde/fast_serde.py

```diff
@@ -0,0 +1,28 @@
+import torch
+import io
+import time
+
+from lmcache.storage_backend.serde.serde import Serializer, Deserializer
+from lmcache.logging import init_logger
+from lmcache.config import GlobalConfig
+
+logger = init_logger(__name__)
+
+class FastSerializer(Serializer):
+    def __init__(self):
+        super().__init__()
+
+    def to_bytes(self, t: torch.Tensor) -> bytes:
+        # FIXME: only support fp16 for now
+        assert t.dtype == torch.float16
```

Comment:
vLLM will use bfloat16 on devices supporting compute compatibility >= 8.0 (including A-series, H-series, L40, etc).
Therefore, it would be very helpful to support flexible data types. 
For instance, maybe we can add a separate tag in the byte stream for the data type?

Reply:
Totally agree. Just to give a rough example here. Let me think how to make it more flexible.